nv2a-trace.py

```diff
@@ -19,7 +19,8 @@
 import Trace
 
 # pylint: disable=invalid-name
-_enable_experimental_disable_z_compression_and_tiling = True
+# TODO: Remove tiling suppression once AGP read in Texture.py is fully proven.
+_enable_experimental_disable_z_compression_and_tiling = False
```

Comment:
I'd rather have this proven to work and then remove this code path entirely?
It makes the capturing much less invasive and some rather troublesome dead code could be emulated (seeing how capturing must be started twice, once to disable tiling, and then again to actually capture untiled memory once the GPU has actually disabled tiling on the next frames - some buffers might remain corrupted forever).